web/cobrands/buckinghamshire/assets.js

```diff
@@ -4,6 +4,30 @@ if (!fixmystreet.maps) {
     return;
 }
 
+// ArcGIS wants to receive the bounding box as a 'geometry' parameter, not 'bbox'
+var format = new OpenLayers.Format.QueryStringFilter();
+OpenLayers.Protocol.BuckinghamshireHTTP = OpenLayers.Class(OpenLayers.Protocol.HTTP, {
+    filterToParams: function(filter, params) {
+        params = format.write(filter, params);
+        params.geometry = params.bbox;
+        delete params.bbox;
+        return params;
+    },
+    CLASS_NAME: "OpenLayers.Protocol.BuckinghamshireHTTP"
+});
+
+// This is required so that the found/not found actions are fired on category
```

Comment:
Not now, but probably need to at some point think about factoring this out, this type of thing is needed by quite a few now, isn't it - need a nicer way of saying which asset layers need this and which don't.